packages/viewer/tests/cypress/support/commands.ts

```diff
@@ -123,8 +123,7 @@ function goToView(
         withHash = true,
         geolocationMockupOptions = { latitude: 47, longitude: 7 },
         fixturesAndIntercepts = {},
-        legacy = !withHash,
-        firstLoad = true,
```

Comment:
this param might need to stay (maybe Ismail can better judge that), but AFAIK he introduced it one or two PR before to fix some tests

Reply:
What is your other solution, @sommerfe?
I saw you remove the `firstLoad` here, but not in the test (so. the `firstLoad` parameter is not used). Based on what I know, the `gaMapReady` is only sent once after the app is ready.

Strangely, the test is passing in the CI, but in my local there is a failed test
<img width="515" height="395" alt="image" src="https://github.com/user-attachments/assets/7f72c48d-90ef-4961-a649-b34bb80fadb6" />